src/lib/components/intl-tel-input.component.ts

```diff
@@ -26,19 +24,22 @@ export class IntlTelInputComponent implements AfterViewInit {
     @Input() label!: string;
     @Input() labelCssClass!: string;
     @Input() name = 'intl-tel-input-name';
-    @Input() options: IntlTelInputOptions = {};
+    @Input() options:  SomeOptions = {};
     @Input() required!: boolean;
+    @Input() isMobileOnly: boolean = true;
```

Comment:
This should go in a dedicated commit, and tets should be added.Also, I think that it should be false by default to be the least breaking as possible